orchestrator/test_runner/src/erc_721_happy_path.rs

```diff
@@ -0,0 +1,638 @@
+                if let Some(expected) = expected_change.clone() {
+                    if end_coin.amount.clone() - start_coin.amount.clone() == expected
+                        && start_coin.denom == end_coin.denom
+                    {
+                        info!(
+                            "Successfully bridged ERC20 {}{} to Cosmos! Balance is now {}{}",
+                            amount, start_coin.denom, end_coin.amount, end_coin.denom
+                        );
+                        return Ok(());
+                    }
+                } else {
+                    let expected_end = start_coin.amount.checked_add(&amount.clone());
+                    if expected_end.is_none() {
+                        info!(
+                            "Expecting overflow from addition of {:?} + {:?}!",
+                            start_coin.amount,
+                            amount.clone()
+                        );
+                    } else if start_coin.amount + amount.clone() == end_coin.amount
+                        && start_coin.denom == end_coin.denom
+                    {
+                        info!(
+                            "Successfully bridged ERC20 {}{} to Cosmos! Balance is now {}{}",
+                            amount, start_coin.denom, end_coin.amount, end_coin.denom
+                        );
+                        return Ok(());
+                    }
+                }
+            }
+            (None, Some(end_coin)) => {
+                // When a bridge governance vote happens, the orchestrator will replay all incomplete
+                // sends to cosmos on the next send to cosmos transaction, so we need to use expected_change
+                if let Some(expected) = expected_change.clone() {
+                    if end_coin.amount == expected {
+                        info!(
+                            "Successfully bridged ERC20 {}{} to Cosmos! Balance is now {}{}",
+                            amount, end_coin.denom, end_coin.amount, end_coin.denom,
+                        );
+                        return Ok(());
+                    }
+                } else if amount == end_coin.amount {
+                    info!(
+                        "Successfully bridged ERC20 {}{} to Cosmos! Balance is now {}{}",
+                        amount, end_coin.denom, end_coin.amount, end_coin.denom
+                    );
+                    return Ok(());
+                } else {
+                    panic!("Failed to bridge ERC20!")
+                }
+            }
+            _ => {}
+        }
+        info!("Waiting for ERC20 deposit");
+        contact.wait_for_next_block(TOTAL_TIMEOUT).await.unwrap();
+    }
+    Err(GravityError::InvalidBridgeStateError(
+        "Did not complete deposit!".to_string(),
+    ))
+}
+
+// Tries up to TOTAL_TIMEOUT time to find a MsgSendToCosmosClaim attestation created in the
+// test_erc20_deposit test
+async fn check_send_to_cosmos_attestation(
```

Comment:
this needs to be modified to locate the GravityERC721 attestation.

Reply:
Not sure what testing we are doing regarding attestations, will need to discuss